packages/tldraw/src/lib/ui/components/primitives/menus/TldrawUiMenuActionCheckboxItem.tsx

```diff
@@ -0,0 +1,21 @@
+import { useActions } from '../../../context/actions'
+import {
+	TldrawUiMenuCheckboxItem,
+	type TLUiMenuCheckboxItemProps,
+} from './TldrawUiMenuCheckboxItem'
+
+/** @public */
+export type TLUiMenuActionCheckboxItemProps = {
+	actionId?: string
+} & Pick<TLUiMenuCheckboxItemProps, 'disabled' | 'checked' | 'toggle'>
```

Comment:
We could spread all of the props, though these are the only ones that are stateful. Maybe there's a better way of typing this to make that more apparent?